code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -426,16 +419,6 @@
 				M.visible_message("<span class='nicegreen'>You successfully replace most of the burnt off flesh of [M].")
 	..()
 
-/datum/reagent/medicine/synthflesh/on_mob_life(mob/living/carbon/M)
-	M.adjustFireLoss(-0.5*REM, 0)
-	M.adjustBruteLoss(-0.5*REM, 0)
-	..()
-	. = 1
-
-/datum/reagent/medicine/synthflesh/overdose_process(mob/living/M)
-	M.adjustToxLoss(2*REM, 0)
-	..()
-	. = 1
```

Comment:
Nah, synthflesh is already overpowered with the factory. Might as well just let it as it is